webrtc-stats.html

```diff
@@ -1612,6 +1613,21 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>voiceActivityFlag</code></dfn> of type <span class="idlMemberType"><a>
+                boolean</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for audio. Whether the last RTP packet played out by this track
```

Comment:
instead of limiting this to just playout (received streams),  this should be valid for the sent streams as well